common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -228,21 +234,27 @@ object WorkFsm {
     def redirect(redirect: IdState.Identifiable) = redirect
   }
 
-  implicit val mergedToDenormalised = new Transition[Merged, Denormalised] {
-    def state(state: Merged,
-              data: WorkData[DataState.Unidentified],
-              relations: Relations[DataState.Unidentified]): Denormalised =
-      Denormalised(state.sourceIdentifier, state.modifiedTime, relations)
-
-    def data(data: WorkData[DataState.Unidentified]) = data
-
-    def redirect(redirect: IdState.Identifiable) = redirect
-  }
+  implicit val identifiedToDenormalised =
```

Comment:
why there's no `mergedToIdentified`?

Reply:
ah ignore me. Because the idminter is work model agnostic

Reply:
Because the `id_minter` does that transition, and its magical / untyped so doesnt need a `Transition[_, _]`

Reply:
Yup, snap